src/pages/_sections/Information.tsx

```diff
@@ -48,6 +48,11 @@ export function Information() {
           ))}
         </ul>
       </div>
+      <SimpleCard>
+        <section className="mx-auto">
+          <Countdown />
+        </section>
+      </SimpleCard>
```

Comment:
```suggestion
      <SimpleCard className="mx-auto flex flex-wrap justify-center items-center gap-24">
          <Countdown />
      </SimpleCard>
```

Lo que era le container en el componente pasaría a estar acá.